upup/models/cloudup/resources/addons/aws-load-balancer-controller.addons.k8s.io/k8s-1.19.yaml.template

```diff
@@ -749,8 +749,8 @@ spec:
       containers:
       - args:
         - --cluster-name={{ ClusterName }}
-        - --enable-waf=false
-        - --enable-wafv2=false
+        - --enable-waf={{ WithDefaultBool .AWSLoadBalancerController.EnableUseOfWAFClassic false }}
```

Comment:
I prefer that the spec field and flag match. Any reason they shouldn't?

Reply:
I found that the flag names didn't match AWS' language about these services or resources (e.g. the `--enable-waf` flag controls use of WAF Classic), so I tried to recast the field names in terms that users would confront in the AWS documentation.

However, if you see the _Cluster_ spec as being a summary of configuration that the users would have written themselves in a manifest like this one, I can accept your preference as being the appropriate choice.

Reply:
By convention, we try to match flags. It can get awkward when upstream is not consistent, but people tend to look for "how do I enable this flag" rather than "how do I enable this feature". In my humble experience.